locales/en/messages.json

```diff
@@ -14,6 +14,15 @@
         "message": "No",
         "description": "General No message to be used across the application"
     },
+    "on": {
+        "message": "On"
```

Comment:
We already have 'On', 'Off', and 'Auto' multiple times in the messages. We should replace these with the new generic messages as well.

Not sure if this should be done now, or if we want to wait and do it after 10.6 has been released, but it should be done.

Reply:
Yeah, there are a lot of messages of this kind. Like save button, refresh are set for each tab individually and it would be best to convert all of them to a single generic message. Probably best to leave it for after 10.6 is out.

Reply:
We should probably add a check at the same time that fails pull requests in CI if they attempt to add a message that is identical to an existing one...